kubernetes/client/clientset/typed/compose/v1beta2/stack.go

```diff
@@ -0,0 +1,155 @@
+package v1beta2
+
+import (
+	scheme "github.com/docker/cli/kubernetes/client/clientset/scheme"
+	v1beta2 "github.com/docker/cli/kubernetes/compose/v1beta2"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	watch "k8s.io/apimachinery/pkg/watch"
+	rest "k8s.io/client-go/rest"
+)
+
+// StacksGetter has a method to return a StackInterface.
+// A group's client should implement this interface.
+type StacksGetter interface {
+	Stacks(namespace string) StackInterface
+}
+
+// StackInterface has methods to work with Stack resources.
+type StackInterface interface {
+	Create(*v1beta2.Stack) (*v1beta2.Stack, error)
+	Update(*v1beta2.Stack) (*v1beta2.Stack, error)
+	UpdateStatus(*v1beta2.Stack) (*v1beta2.Stack, error)
+	Delete(name string, options *v1.DeleteOptions) error
+	DeleteCollection(options *v1.DeleteOptions, listOptions v1.ListOptions) error
+	Get(name string, options v1.GetOptions) (*v1beta2.Stack, error)
+	List(opts v1.ListOptions) (*v1beta2.StackList, error)
+	Watch(opts v1.ListOptions) (watch.Interface, error)
+	Patch(name string, pt types.PatchType, data []byte, subresources ...string) (*v1beta2.Stack, error)
+}
+
+// stacks implements StackInterface
+type stacks struct {
+	client rest.Interface
+	ns     string
+}
+
+// newStacks returns a Stacks
+func newStacks(c *ComposeV1beta2Client, namespace string) *stacks {
+	return &stacks{
+		client: c.RESTClient(),
+		ns:     namespace,
+	}
+}
+
+// Create takes the representation of a stack and creates it.  Returns the server's representation of the stack, and an error, if there is any.
+func (c *stacks) Create(stack *v1beta2.Stack) (*v1beta2.Stack, error) {
+	result := &v1beta2.Stack{}
+	err := c.client.Post().
+		Namespace(c.ns).
+		Resource("stacks").
+		Body(stack).
+		Do().
+		Into(result)
+	return result, err
+}
+
+// Update takes the representation of a stack and updates it. Returns the server's representation of the stack, and an error, if there is any.
+func (c *stacks) Update(stack *v1beta2.Stack) (*v1beta2.Stack, error) {
+	result := &v1beta2.Stack{}
+	err := c.client.Put().
+		Namespace(c.ns).
+		Resource("stacks").
+		Name(stack.Name).
+		Body(stack).
+		Do().
+		Into(result)
+	return result, err
+}
+
+// UpdateStatus was generated because the type contains a Status member.
+// Add a +genclientstatus=false comment above the type to avoid generating UpdateStatus().
```

Comment:
Looks like `UpdateStatus()` isn't used? should we remove this? (same for the other API version)